### PR TITLE
Use Bitcoin Gold instead of Core in configure.ac

### PR DESCRIPTION
**Pull request by TheSorrow1**
Changed name for the branding,

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -8,7 +8,7 @@ define(_CLIENT_VERSION_IS_RELEASE, true)
 define(_COPYRIGHT_YEAR, 2017)
 define(_COPYRIGHT_HOLDERS,[The %s developers])
 define(_COPYRIGHT_HOLDERS_SUBSTITUTION,[[Bitcoin Core]])
```
Comment (h4x3rotab) [inline configure.ac]: Maybe also change this line? Let's see how others make changes.

File: configure.ac [hunk 2]
```diff
@@ -8,7 +8,7 @@ define(_CLIENT_VERSION_IS_RELEASE, true)
...
-AC_INIT([Bitcoin Core],[_CLIENT_VERSION_MAJOR._CLIENT_VERSION_MINOR._CLIENT_VERSION_REVISION],[https://github.com/bitcoin/bitcoin/issues],[bitcoin],[https://bitcoincore.org/])
+AC_INIT([Bitcoin Gold],[_CLIENT_VERSION_MAJOR._CLIENT_VERSION_MINOR._CLIENT_VERSION_REVISION],[https://github.com/bitcoin/bitcoin/issues],[bitcoin],[https://bitcoincore.org/])
```
Comment (h4x3rotab) [inline configure.ac]: lgtm. Should also change the link to the repo and website when ready.

File: configure.ac (hunk 1 above)
Comment (TheSorrow1) [reply configure.ac]: Yes,

File: configure.ac (hunk 2 above)
Comment (TheSorrow1) [reply configure.ac]: The only reason I didn't change the links are beacuse we don't have a finished website yet.

**Pull request closed without merging**